remarks/remarks.py

```diff
@@ -39,153 +41,158 @@ def run_remarks(
     combined_pdf=False,
     modified_pdf=False,
 ):
-    for path in pathlib.Path(f"{input_dir}/").glob("*.pdf"):
-        pages = list_pages_uuids(path)
-        name = get_visible_name(path)
-        rm_files = list_ann_rm_files(path)
+    for path in pathlib.Path(f"{input_dir}/").glob("*.metadata"):
+        if is_document(path):
```

Comment:
@clement-elbaz, quick observation – 

What do you think of having instead: 

```python
if not is_document(path):
    continue
```

This way we could avoid one tab (shift).

Thanks!

Reply:
Thank you! This makes sense. I just force-pushed my previous commit to update this. Strangely this has made the diff uglier though: neither git-diff, nor meld nor the Github difftool manages to display it neatly. Compared to my previous commit the only thing I added is this change.